src/polar/types.ts

```diff
@@ -0,0 +1,10 @@
+import { Props as PolarAngleAxisProps } from './PolarAngleAxis';
+import { Props as PolarRadiusAxisProps } from './PolarRadiusAxis';
+
+// TODO: 1. what is the type of scale? 2. Should we include scale to each Props?
+export type AngleAxisProps = PolarAngleAxisProps & { scale: any };
```

Comment:
There is a cycle-import here. `PolarAngleAxis` import  some attributes from `types.ts`.